frontend/theta-project-tracker/src/app/services/timsheet.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+import { Observable } from 'rxjs';
+import { RecordCreate } from '../models/record-create.model';
+
+@Injectable({ providedIn: 'root' })
+export class TimesheetService {
+    apiUrl: string = environment.baseUrl;
+    constructor(private http: HttpClient) { }
+
+    getTimeRecords(date?: string): Observable<TimeRecordResponse> {
+        console.log('service', date)
+        return this.http.get<TimeRecordResponse>(this.apiUrl + `timeRecord?date=${date}`);
```

Comment:
ezek is query paramok?

Reply:
javítottam, viszont a PUT csak úgy fogadott el mindent, ha kitöröltem a visszatérési értékét